ankidroid/Anki-Android
# Make icons colors more consistent and add icons to the theme preferences

Was doing something else and got the desire to add icons to the theme preferences. Fixed/cleaned some things while doing it

## Approach
1. Set preference headers tint on the drawable XMLs instead of programatically
2. Set the "Language" preference icon to use the same default icon color
3. Added some icons for the theme preferences

## How Has This Been Tested?

SDK 29 emulator

1. Go to settings > See if the icon colors are correct

![image](https://github.com/ankidroid/Anki-Android/assets/69634269/ab5e4421-6dcf-4c3b-a19f-db08e5ecf313)

2. See if there wasn't a problem with  the reviewer icons (specifically the Whiteboard editor icon)

![230515162643-Anki-Android_–_MenuUtils kt_ Anki-Android AnkiDroi](https://github.com/ankidroid/Anki-Android/assets/69634269/b27553cf-c0b1-4011-87f4-b620278f89a2)

3. Settings > General > See if the language icon color is correct
![230515162603-Anki-Android_–_MenuUtils kt_ Anki-Android AnkiDroi](https://github.com/ankidroid/Anki-Android/assets/69634269/c12acb62-eb8e-47b3-ba8f-98c4614f8df0)

4. Settings > Appearance > See the new theme icons
![230515162626-Anki-Android_–_MenuUtils kt_ Anki-Android AnkiDroi](https://github.com/ankidroid/Anki-Android/assets/69634269/757322eb-9cc7-44e8-8101-f72fe9d87aa4)


## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._
